packages/api/src/types.ts

```diff
@@ -4,7 +4,7 @@ import { ModerationPrefs } from './moderation/types'
 /**
  * Supported proxy targets
  */
-export type AtprotoServiceType = 'atproto_labeler'
+export type AtprotoServiceType = string // 'atproto_labeler'
```

Comment:
Was this change intended to stick around?  Maybe we can split the difference and go with something like `'atproto_labeler' | (string & {})`.

Reply:
I don't remember exactly why I did this. I am 95% sure that if I did this, it must have been because I got an error somewhere (ozone repo?).

I'm in favor of reverting it if we can't find the source.